RedHatInsights/insights-remediations-frontend
# fix(RHIF-266): adopt useChrome updateDocumentTitle hook

# Description
A partial cause of https://issues.redhat.com/browse/RHIF-207 is that the document title isnt updated properly. This PR introduces the chrome hook.
Associated Jira ticket: # (RHIF-266)

# How to test the PR
Navigate to the home page and the details page and verify that the document changes appropriately 


# Checklist:
